src/pathways/pathway/pathways.service.ts

```
@@ -2423,6 +2425,37 @@ export class PathwaysService {
 
+      // Send course-completion email for the completed VOLUNTEER pathway.
+      try {        
+        const user = await this.userRepository.findOne({ where: { userId: dto.userId } });
+        if (user?.email) {
+          const pathwayName = record.pathway.name;
+          await this.notificationRequest.sendNotification({
+            isQueue: false,
+            context: 'USER',
+            key: 'onPathwayCompletion',
+            replacements: {
+              '{username}': `${user.firstName} ${user.lastName}`.trim(),
```

ESLint

[error] Replace `'{username}'` with `"{username}"`

(prettier/prettier)